onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -14,6 +14,100 @@
 from .registry import IntegerOpsRegistry, QLinearOpsRegistry
 
 
+class QuantConfig:
```

Comment:
Could you please also add a config for Dynamic quantization? And also add default config for eps: CPU, TRT, NNAPI?